scripts/dist.ts

```diff
@@ -78,6 +78,8 @@ function toDist(sourcePath: string): YAML.Document {
   );
   const { name: id } = path.parse(sourcePath);
 
+  const entrypoint: yaml.get('entrypoint');
```

Comment:
For my earlier suggestion, instead we'd have to check whether `compute_from` is set. If it is, we'd treat it as `overridden.compatFeatures` instead of `compat_features` itself.